src/main/java/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int COLOR_COUNT = 8;
+    public static final int MAX_SIDE = 50;
+    private Random random = new Random();
+
+    // Why not to cut this decision? ->
+    // Color of figure is a state of each and every figure,
+    // so it can be seen as something, that exists beneath figure,
+    // and it's possible to include ColorSupplier as a subclass of FigureSupplier.
+    // Or not?
+    // Also, we don't create redundant files.
```

Comment:
I think it's okay to do it like this here, but next time let's stick to the task description. It's important to learn the basics and how to write quality code first. Then we'll learn how to improve it :)

P.s.
Let's remove this comment from the code